<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pizzas MTI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1218</width>
     <height>729</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1218</width>
     <height>729</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupHilos">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lblCount">
       <property name="geometry">
        <rect>
         <x>1080</x>
         <y>30</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { color : white; }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblTiempoTranscurrido">
       <property name="geometry">
        <rect>
         <x>1070</x>
         <y>10</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { color : white; }</string>
       </property>
       <property name="text">
        <string>Tiempo transcurrido</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QTableWidget" name="tblPedidos">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>150</y>
         <width>101</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="autoScrollMargin">
        <number>16</number>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <column/>
       <column/>
      </widget>
      <widget class="QLabel" name="imgBackRestaurant">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>651</width>
         <height>711</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/back_restaurante.jpg</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="pizza01">
       <property name="geometry">
        <rect>
         <x>-30</x>
         <y>450</y>
         <width>311</width>
         <height>291</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/pizza/pizza_0.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="pizza02">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>450</y>
         <width>311</width>
         <height>291</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/pizza/pizza_0.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="pizza03">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>450</y>
         <width>311</width>
         <height>291</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/pizza/pizza_0.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="imgBackStreet">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>0</y>
         <width>561</width>
         <height>711</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/back_street.jpg</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/logo.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="imgRepartidor1">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>320</y>
         <width>431</width>
         <height>391</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/repartidor-1.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="imgRepartidor2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>340</y>
         <width>410</width>
         <height>391</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/repartidor-2.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="imgPhone">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>100</y>
         <width>260</width>
         <height>321</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/phone.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>0</y>
         <width>16</width>
         <height>711</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/linea_separacion.jpg</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="imgPizzaBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>251</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/pizza_box.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="img_num_pedidos">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/img_num_pedidos.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="lblNoPedidos">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>70</y>
         <width>61</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/img_pizzas-lista.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>101</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/img_num_pizzas_listas.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="lblNoListos">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>150</y>
         <width>91</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblNoPedido1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>420</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblNoPedido3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>420</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblNoPedido2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>430</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>200</y>
         <width>121</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/phone_02.png</pixmap>
       </property>
      </widget>
      <widget class="QProgressBar" name="pbRepartidor02">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>70</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {border: 2px solid grey;border-radius: 5px;background-color: #fbf3ec;}QProgressBar::chunk {background-color: #99d4d9;width: 20px;}</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string> %v seg  </string>
       </property>
      </widget>
      <widget class="QProgressBar" name="pbRepartidor01">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>30</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {border: 2px solid grey;border-radius: 5px;background-color: #fbf3ec;}QProgressBar::chunk {background-color: #daa482;width: 20px;}</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="format">
        <string> %v seg  </string>
       </property>
      </widget>
      <widget class="QLabel" name="imgRepartidorIcon1">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>20</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/repartidor-1_icon.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="imgRepartidorIcon2">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>assets/repartidor-2_icon.png</pixmap>
       </property>
      </widget>
      <widget class="QProgressBar" name="pbCocinero02">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>680</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {border: 2px solid grey;border-radius: 5px;background-color: #fbf3ec;}QProgressBar::chunk {background-color: #daa482;width: 20px;}</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="format">
        <string> %v seg / Coc 2   </string>
       </property>
      </widget>
      <widget class="QProgressBar" name="pbCocinero01">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>680</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {border: 2px solid grey;border-radius: 5px;background-color: #fbf3ec;}QProgressBar::chunk {background-color: #99d4d9;width: 20px;}</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="format">
        <string> %v seg / Coc 1  </string>
       </property>
      </widget>
      <widget class="QProgressBar" name="pbCocinero03">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>680</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {border: 2px solid grey;border-radius: 5px;background-color: #fbf3ec;}QProgressBar::chunk {background-color: #0edb0e;width: 20px;}</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="format">
        <string> %v seg / Coc 3  </string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRepartidor1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>30</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRepartidor2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>70</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCocinero1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>680</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCocinero2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>680</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCocinero3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>680</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCrearPedido">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>1090</x>
         <y>90</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>assets/ico_pedido.png</normaloff>assets/ico_pedido.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPedidos">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>70</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>1090</x>
         <y>70</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { color : white; }</string>
       </property>
       <property name="text">
        <string>Pedidos</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblNoPedidosCompletos">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>61</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblNoPedidos_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>51</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblPedidoRepartiendo1">
       <property name="geometry">
        <rect>
         <x>706</x>
         <y>10</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblPedidoRepartiendo2">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>50</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <zorder>imgBackStreet</zorder>
      <zorder>imgRepartidor1</zorder>
      <zorder>imgRepartidor2</zorder>
      <zorder>imgBackRestaurant</zorder>
      <zorder>lblTiempoTranscurrido</zorder>
      <zorder>lblCount</zorder>
      <zorder>pizza01</zorder>
      <zorder>pizza02</zorder>
      <zorder>pizza03</zorder>
      <zorder>label_3</zorder>
      <zorder>imgPhone</zorder>
      <zorder>imgPizzaBox</zorder>
      <zorder>img_num_pedidos</zorder>
      <zorder>lblNoPedidos</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>lblNoListos</zorder>
      <zorder>label_11</zorder>
      <zorder>lblNoPedido1</zorder>
      <zorder>lblNoPedido3</zorder>
      <zorder>lblNoPedido2</zorder>
      <zorder>tblPedidos</zorder>
      <zorder>label_14</zorder>
      <zorder>label_4</zorder>
      <zorder>pbRepartidor02</zorder>
      <zorder>pbRepartidor01</zorder>
      <zorder>imgRepartidorIcon2</zorder>
      <zorder>imgRepartidorIcon1</zorder>
      <zorder>pbCocinero02</zorder>
      <zorder>pbCocinero01</zorder>
      <zorder>pbCocinero03</zorder>
      <zorder>btnRepartidor1</zorder>
      <zorder>btnRepartidor2</zorder>
      <zorder>btnCocinero1</zorder>
      <zorder>btnCocinero2</zorder>
      <zorder>btnCocinero3</zorder>
      <zorder>btnCrearPedido</zorder>
      <zorder>txtPedidos</zorder>
      <zorder>label</zorder>
      <zorder>lblNoPedidosCompletos</zorder>
      <zorder>lblNoPedidos_3</zorder>
      <zorder>lblPedidoRepartiendo1</zorder>
      <zorder>lblPedidoRepartiendo2</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
